### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by ghost** [NationalSecurityAgency/ghidra#3291] [General]
I just installed the new version, I can't run the application. Please help. Thanks. The old version is fine.

**Comment by topia** [comment]
> I just installed the new version, I can't run the application. Please help. Thanks. The old version is fine.

Need more info. What is your OS? JVM version? What version is the old/new Ghidra?
For a recent frequently asked problem, using Windows and installing Ghidra into the path contains space?

**Reply by ghost** [reply]
[
![Screenshot_1](https://user-images.githubusercontent.com/77043453/127805931-bf50199b-cb39-4fa2-b8af-5d3c38fcb7f7.png)
![Screenshot_2](https://user-images.githubusercontent.com/77043453/127805938-60b876c4-3cb2-49ec-a226-fbc183e24e5e.png)
![Screenshot_19](https://user-images.githubusercontent.com/77043453/127805944-b5002288-a746-4c72-af4e-942e7e82855d.png)
](url)

**Reply by ryanmkurtz** [reply]
Do you have a space in your Ghidra installation directory path?

**Reply by ghost** [reply]
> Do you have a space in your Ghidra installation directory path?

I do not get what you mean.

**Reply by ryanmkurtz** [reply]
I am asking about where you have Ghidra installed.  For example, you might have it installed at `c:\Ghidra` or `c:\folder with spaces\Ghidra`.

**Reply by ghost** [reply]
Let's me try.

**Reply by ghost** [reply]
> I am asking about where you have Ghidra installed. For example, you might have it installed at `c:\Ghidra` or `c:\folder with spaces\Ghidra`.

Still can't.

**Reply by ryanmkurtz** [reply]
If you are Windows, and Ghidra 10.0 runs but Ghidra 10.0.1 does not, it is because you put your Ghidra in a folder with spaces in it.  Try installing Ghidra at `c:\ghidra` and running it from there.  If that does not work, please include any error messages that you see.

**Reply by ghost** [reply]
> If you are Windows, and Ghidra 10.0 runs but Ghidra 10.0.1 does not, it is because you put your Ghidra in a folder with spaces in it. Try installing Ghidra at `c:\ghidra` and running it from there. If that does not work, please include any error messages that you see.

I so many thank . I got it.

**Reply by ghost** [reply]
> If you are Windows, and Ghidra 10.0 runs but Ghidra 10.0.1 does not, it is because you put your Ghidra in a folder with spaces in it. Try installing Ghidra at `c:\ghidra` and running it from there. If that does not work, please include any error messages that you see.

I so many thank . I got it.

**Reply by ghost** [reply]
> I am asking about where you have Ghidra installed. For example, you might have it installed at `c:\Ghidra` or `c:\folder with spaces\Ghidra`.

I can't setup Grassmarlin. Plz help me.

**Reply by ryanmkurtz** [reply]
We won't be able to help you with setting up Grassmarlin here....sorry!

**Reply by ghost** [reply]
> We won't be able to help you with setting up Grassmarlin here....sorry!

Thank you very much.
